### ISSUE TITLE
How can I add a babel plugin?

### ISSUE THREAD

**Issue by wadim** [rocjs/roc-package-web-app-react#26]
I would like to add the `transform-es3-member-expression-literals` plugin to the babel transform. Is it possible to add this somewhere?

**Comment by dlmr** [comment]
Yes! 

The easiest way is to create a `.babelrc` that will be used by Roc.

http://babeljs.io/docs/usage/babelrc/

**Comment by wadim** [comment]
I placed a `.babelrc` file both inside `src` and inside the package root. Babel does not seem to load the plugin though.

The file below is not a full `.babelrc` file - it does not contain all other necessary transformations - but I added it as a test to see if babel would recognise it at all.

```
{
  "plugins": ["transform-es3-member-expression-literals"]
}
```

Babel works as before and the transform is not employed.

**Comment by dlmr** [comment]
Strange... Another way of doing it is to add the plugin directly in the Webpack configuration.

This can be done in the following way at the moment. 

``` javascript
// in your roc.config.js
{
   action: () => (rocObject) => {
        const hook = rocObject.hook;
        const rocBuilder = rocObject.previousValue;
        if (hook === 'build-webpack') {
            return () => () => {
                rocBuilder.buildConfig.babel.plugins.push(require.resolve('babel-plugin-transform-es3-member-expression-literals'));
                return rocBuilder;
            }
        }
    }
}
```

**Comment by wadim** [comment]
I tried that exact code too but that did not help solve the problem.

However I found out that loading the plugin actually works. It's just that **some** modules within the bundle still had references to `.default` (which is what the transformation is about). The ones which still had it seemed to come from `babel-runtime` itself (not from any code transpiled by babel). I cannot confirm that with 100% certainty at the moment though.

Anyway, I found out that I don't actually need to support browsers which would necessitate this transformation so this issue is no longer relevant and can be closed for my sake.

Thanks for the help, @dlmr!
